src/controller/controller.js

```diff
@@ -0,0 +1,59 @@
+const model = require("../models/user");
+
+//  Add new data
+exports.adddata = async (req, res) => {
+  try {
+    await model.create(({ userName, email, birthdate, age, password } = req.body));
```

Comment:
shorthand method used and remove extra variable and direct passed the value for both adddata and update data 

await model.create(({ userName, email, birthdate, age, password } = req.body));

Reply:
not like that

Reply:
`const { name, email, age, birthdate, password } = req.body;`

    await model.create({
      name,
      email,
      age,
      birthdate,
      password,
    });`